Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.21 AS builder
+
+WORKDIR /building
+COPY . .
+
+ARG APP
+RUN make ${APP}
+
+FROM alpine:3.18 AS runtime
```

Comment:
Before, it was `FROM alpine:3`?

Reply:
It' best practice to use specific versions, and it's effectively the same version (at the moment).
So if a project decides to improve security by rebuilding images even without code change, then version numbers make it easier for humans to accept the suggestions dependabot makes.
https://github.com/hadolint/hadolint/wiki/DL3007